### ISSUE TITLE
Build error when call Injector.InjectViews() inside fragment

### ISSUE THREAD

**Issue by MAX-POLKOVNIK** [MAX-POLKOVNIK/DroidInjector#1]
```
Fody: An unhandled exception occurred:
Exception:
Value cannot be null.
Parameter name: method
StackTrace:
   at Mono.Cecil.Cil.Instruction.Create(OpCode opcode, MethodReference method) in C:\projects\cecil\Mono.Cecil.Cil\Instruction.cs:line 174
   at Polkovnik.DroidInjector.Fody.InjectorCallReplacer.ReplaceParameterlessMethodInstructions(Instruction callInjectorInstruction, ILProcessor ilProcessor, MethodReference injectionMethod, MethodReference getViewMethod)
   at Polkovnik.DroidInjector.Fody.InjectorCallReplacer.Execute()
   at Polkovnik.DroidInjector.Fody.FodyInjector.Execute()
   at Polkovnik.DroidInjector.Fody.ModuleWeaver.Execute()
   at lambda_method(Closure , Object )
   at InnerWeaver.ExecuteWeavers()
   at InnerWeaver.Execute()
Source:
Mono.Cecil
TargetSite:
Mono.Cecil.Cil.Instruction Create(Mono.Cecil.Cil.OpCode, Mono.Cecil.MethodReference)
```

**Comment by MAX-POLKOVNIK** [comment]
Need to call Injector.InjectViews(View view);

I will add more detailed message in output.

**Comment by MAX-POLKOVNIK** [comment]
Fixed in 0.1.4
